chart/compass/values.yaml

```diff
@@ -2,12 +2,15 @@ global:
   images:
     containerRegistry:
       path: eu.gcr.io/kyma-project/incubator
+    connector:
+      dir: pr/
+      version: "PR-32"
     director:
       dir: pr/
       version: "PR-39"
```

Comment:
Please change the image of Director to `PR-41`. Apparently I forgot to change it 😞

Reply:
Done